docs/manual/developer/11_governance.md

```diff
@@ -86,7 +86,9 @@ All issues and pull requests for product removal must use the [product-removal](
 #### Architectural Decision Records
 - We use [Architectural Decision Records](#introduction-of-architecture-decisions-records)
 - The first ADR defines the context and guidelines for new ADRs.
-
+- All PR that support or justify a proposed ADR should be linked to that ADR in the description and should remain in the draft state until ADR is accepted.
```

Comment:
I imagine cases where a new contributor opens a PR without knowing it could be impacted or related to a proposed ADR. But maintainers would be aware of this during the review. So, we could also include a sentence on how maintainers should proceed in this case. Basically to inform other reviewers and also guide the author through the process.

Reply:
Using dedicated label could be good to easily bring visibility on these PRs.

Reply:
I've added something along the lines.